test/conformance/tests/sriov_operator.go

```diff
@@ -59,6 +60,12 @@ func init() {
 	if err == nil && newTime != 0 {
 		waitingTime = time.Duration(newTime) * time.Minute
 	}
+
+	isSingleNode, err := cluster.IsSingleNode(clients)
```

Comment:
On a second thought, this will break dryRun. Can you move it inside beforeAll or as part of one of the test suite steps?

Reply:
Done